src/ansys/fluent/core/file_session.py

```diff
@@ -31,7 +31,18 @@
 from ansys.fluent.core import PyFluentDeprecationWarning
 from ansys.fluent.core.filereader.case_file import CaseFile
 from ansys.fluent.core.filereader.data_file import DataFile
-from ansys.fluent.core.services.field_data import SurfaceDataType
+from ansys.fluent.core.services.field_data import (
+    FieldDataSource,
+    FieldTransactionSource,
+    PathlinesFieldDataRequest,
+    ScalarFieldDataRequest,
+    SurfaceDataType,
+    SurfaceFieldDataRequest,
+    TransactionFieldData,
+    VectorFieldDataRequest,
+    _AllowedScalarFieldNames,
+    _AllowedSurfaceNames,
+)
```

Comment:
@prmukherj This introduces a coupling. Please place the interfaces in an independent field interfaces module.

Reply:
Done, Thank you.